icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -1512,6 +1512,11 @@ class U_I18N_API NumberFormatterSettings {
      *
      * Pass this method any instance of {@link MeasureUnit}. For units of measure:
      *
+     * <P>
```

Comment:
Here and elsewhere: you don't need the `<p>`s in ICU4C; these comments are rendered as markdown (but you will need them in ICU4J)

Reply:
Done!

Reply:
I don't see the result of "done". Were you going to remove <p> and </p>?

Reply:
+1

Reply:
I am sorry, I forgot to push the changes :D